curious/__init__.py

```diff
@@ -22,19 +22,32 @@ def model_name(model_class):
 
   def __init__(self, model_class, short_name=None):
     self.model_class = model_class
+
     self.model_name = ModelManager.model_name(model_class)
     self.short_name = model_class.__name__ if short_name is None else short_name
+
+    # Relationships/foreign keys have explicit white/blacklists
     self.allowed_relationships = []
     self.disallowed_relationships = []
+
+    # All model fields are included by default, except these
     self.field_excludes = []
+
+    # Fields returned by Curious represented by @properties of the model
+    self.property_fields = []
```

Comment:
It would be more useful to just have a list of attributes/methods to return along with the model. We can check for a callable below and call it if needed.

Reply:
this is a good idea, but extends the scope of the task significantly, as you'd have to do some kind of equivalent idea for "models" that do not inherit from `django.db.Model`. I don't want to extend the scope of this task for the effort, but if you want to add a backlog item, I'd support that.